sys/include/net/dtls/gdtls.h

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (C) 2017 Raul Fuentes <raul.fuentes_samaniego@inria.fr>
```

Comment:
Don't need to put your email here, `Inria` is enough. Your name/emails are written below